portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SyncAttributes.testcase

```diff
@@ -642,18 +642,18 @@ definition {
 		var attributesDataTypeAscending1 = '''
 			male,agreedToTermsOfUse,defaultUser,emailAddressVerified,
 			birthday,createDate,modifiedDate,createDate,modifiedDate,status,classNameId,classPK,contactId,
-			employeeStatusId,parentContactId,prefixListTypeId,suffixListTypeId,companyId,companyId,contactId,facebookId,
-			ldapServerId,portraitId,timeZoneId,userId,emailAddress,employeeNumber,facebookSn,hoursOfOperation,
-			jabberSn,jobClass,skypeSn,smsSn,twitterSn,userName,comments,emailAddress,externalReferenceCode,firstName,
-			firstName,googleUserId,greeting,jobTitle,languageId,lastName,lastName,middleName,middleName,openId
+			employeeStatusId,parentContactId,prefixListTypeId,suffixListTypeId,companyId,contactId,facebookId,
+			ldapServerId,portraitId,timeZoneId,userId,emailAddress,employeeNumber,facebookSn,firstName,hoursOfOperation,
+			jabberSn,jobClass,jobTitle,lastName,middleName,skypeSn,smsSn,twitterSn,userName,comments,emailAddress,externalReferenceCode,
+			firstName,googleUserId,greeting,jobTitle,languageId,lastName,middleName,openId
 		''';
 		var attributesDataTypeAscending2 = "screenName,uuid";
 		var attributesDataTypeDescending1 = '''
-			emailAddress,employeeNumber,facebookSn,hoursOfOperation,
-			jabberSn,jobClass,skypeSn,smsSn,twitterSn,userName,comments,emailAddress,externalReferenceCode,firstName,
-			firstName,googleUserId,greeting,jobTitle,languageId,lastName,lastName,middleName,middleName,openId,screenName,uuid,
+			emailAddress,employeeNumber,facebookSn,firstName,hoursOfOperation,
+			jabberSn,jobClass,jobTitle,skypeSn,smsSn,twitterSn,userName,comments,emailAddress,externalReferenceCode,
+			firstName,googleUserId,greeting,jobTitle,languageId,lastName,middleName,openId,screenName,uuid,
```

Comment:
Hey @joshitagaki, I think that after `jobTitle` it should be `lastName` and after `middleName` and then `skypeSn`. Could you also check and update the list?

![image](https://user-images.githubusercontent.com/76956227/223574788-69527e6f-3b32-45dd-ae30-410677124344.png)